File: 151003/Mytnik/Lab2/Lab2/Lab2/Models/Tweet.cs
using System;
using System.Collections.Generic;
using Lab2.Models;

namespace Lab2;

public partial class Tweet
{
    public int Id { get; set; }

    public int? CreatorId { get; set; }

    public string? Title { get; set; }

    public string? Content { get; set; }

    public string? Created { get; set; }

    public string? Modified { get; set; }

    public virtual Creator? Creator { get; set; }

    public virtual ICollection<Note> Notes { get; set; } = new List<Note>();
}

File: 151004/Stahnov/nosql/publicator/src/main/java/org/education/bean/Tweet.java
package org.education.bean;

import jakarta.persistence.*;
import lombok.*;

import java.time.LocalDateTime;
import java.util.List;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Entity(name = "tbl_tweet")
public class Tweet {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    int id;

    @ManyToOne
    @JoinColumn(name = "creatorId", referencedColumnName = "id")
    Creator creator;

    @ManyToMany(mappedBy = "tweets")
    List<Marker> markers;

    String title;
    String content;
    LocalDateTime created;
    LocalDateTime modified;
}
